pkg/image/definition.go

```diff
@@ -60,6 +60,13 @@ type OperatingSystem struct {
 	Users      []OperatingSystemUser `yaml:"users"`
 	Systemd    Systemd               `yaml:"systemd"`
 	Suma       Suma                  `yaml:"suma"`
+	Packages   Packages              `yaml:"packages"`
+}
+
+type Packages struct {
+	PKGList  []string `yaml:"packageList"`
+	AddRepos []string `yaml:"additionalRepos"`
```

Comment:
The `AddRepos` variable name sounds like a boolean flag so I'd suggest to rename it to `AdditionalRepos` or `AdditionalRepositories`.

Reply:
`AdditionalRepos` gets my vote here. I think the `AddRepos` thing comes from the `zypper` syntax, but I agree that we should clarify this.